docs/guides/specify-runtime-options.mdx

```diff
@@ -0,0 +1,310 @@
+
+Options are specified differently depending on whether you are using the V1 or V2 interface.
+
+## V2 primitives
+
+Options are specified differently in the V2 primitives in these ways:
+
+- `SamplerV2` and `EstimatorV2` have separate options classes. You can see the available options and update option values during or after primitive initialization.
+- Instead of the `set_options()` method, V2 primitive options have the `update()` method that applies changes to the `options` attribute.
+- If you do not specify a value for an option, it is given a special value of `Unset` and the server defaults are used.
+- For V2 primitives, the `options` attribute is the `dataclass` Python type.  You can use the built-in `asdict` method to convert it to a dictionary.
+
+## Instantiate the Options class (V1)
+
+When using V1 primitives, you must Instantiate the options class before you can set the options. In the example below, we create an instance of the `Options` class. `optimization_level` is a first-level option and can be passed as an input parameter. Options related to the execution environment are passed using the `environment` parameter.
+
+```python
+from qiskit_ibm_runtime import Options
+
+options = Options(optimization_level=1, environment={"log_level": "INFO"})
+```
+
+The `Options` class supports auto-complete. Once you create an instance of the `Options` class, you can use auto-complete to see what options are available. If you choose one of the categories, you can use auto-complete again to see what options are available under that category.
+
+```python
+from qiskit_ibm_runtime import Options
+
+options = Options()
+options.resilience_level = 1
+options.execution.shots = 2048
+```
+<span id="pass-options"></span>
+## Set primitive options
+
+You can set options when initializing the primitive, after initializing the primitive, or in the `run()` method. See the [precidence rules](runtime-options-overview#options-precedence) section to understand what happens when the same option is specified in multiple places.
+
+### Primitive initialization
+
+You can pass in an instance of the options class or a dictionary when initializing a primitive.
+
+#### Options class
+
+When creating an instance of the `Estimator` or `Sampler` class, you can pass in the `options` you created in the options class. Those options will then be applied when you use `run()` to perform the calculation.  Specify the options in this format:  `primitive.options.option.sub-option.sub-sub-option = choice`.  For example: `estimator.options.dynamical_decoupling.enable = True`
+
+Example:
+
+<Tabs>
+  <TabItem value="PrimV2" label="V2 primitives">
+    `SamplerV2` and `EstimatorV2` have separate options classes ([`EstimatorOptions`](/api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.EstimatorOptions) and [`SamplerOptions`](/api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.SamplerOptions)) that do not need to be instantiated.
```

Comment:
Those classes don't need to be instantiated, but they can if you want to pass it in during primitive initialization. The code example below should be something like (mimicking V1 example)

```python
from qiskit_ibm_runtime.options import EstimatorOptions

options =  EstimatorOptions(resilience_level=2, resilience={"zne_mitigation": True, "zne": {"noise_factors": [1, 3, 5]}})

# or...
options =  EstimatorOptions()
estimator.options.resilience_level = 2
options.resilience.zne_mitigation = True
options.resilience.zne.noise_factors = [1, 3, 5]

estimator = Estimator(mode=backend, options=options)
```

Reply:
Thanks!  I knew I'd seen the class bein instantiated somewhere but couldn't find it!